marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/command/AbstractInstallerCommand.java

```diff
@@ -81,4 +84,21 @@ protected String getServerMajorVersion() {
             return "9";
         }
     }
+
+    protected ObjectNode canInstallDhs(String installedHubVersion, MarkLogicVersion mlVersion){
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode node = mapper.createObjectNode();
+        if(Character.getNumericValue(installedHubVersion.charAt(0)) < 5){
```

Comment:
A scenario I forgot in the story is - DHF isn't installed yet. Thus, we should have a try/catch on getInstalledVersion(), so that if we get an exception, we know DHF isn't installed, and thus we don't need to check anything here.